### DISCUSSION TITLE
Load profile based on path

### DISCUSSION THREAD

**Discussion by jesusfj710** [envio-cli/envio#44] [Q&A] [answered]
Hello there,

This tool is really cool. Thanks for the hard work here.

I wonder if there is a solution or a way to configure it depending on the path you are on in the terminal. I didn't find anything in the documentation and I think it would be super useful. Have for example a work profile in /home/work and a personal profile in /home/personal and have them load dynamically depending on where you are.

Thanks

**Comment by humblepenguinn** [comment, marked as answer]
Hi @jesusfj710 thank you for using `envio`!

Dynamic loading is something I'm. Currently working on and will be apart of the next release.

We're going to achieve this by using shell hooks.

**Reply by jesusfj710** [reply]
That's wonderful. Thanks for the quick answer :D
